src/pages/accounts/EmailForm.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { Button } from './AccountRouter.js';
+const img =
+  'https://res.cloudinary.com/feyton/image/upload/v1645616111/Codebandits/Phantom_icon_1_tpjkws.png';
+const EM_RGX = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+function ResetFormPage() {
+  let navigate = useNavigate();
+  const [email, setemail] = useState('');
+  const [emailValid, setemailValid] = useState(false);
+  const [emailFocus, setemailFocus] = useState(false);
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+    const email = e.target.email.value;
+    if (email == 'fab@me.com') {
+      toast('You account was found');
+      navigate('/accounts/reset-email/abcd-1234-ghyi-567/123456');
+    } else {
+      toast('Account not found');
+    }
+  };
+  useEffect(() => {
+    const result = EM_RGX.test(email);
+    setemailValid(result);
+  }, [email]);
+  return (
+    <>
+      <h3 className="text-2xl font-bold text-center py-3 font-rale">
+        RESET PASSWORD
+      </h3>
+      <p className="mb-3">Enter your email to receive a password reset link</p>
+      <form onSubmit={onSubmit}>
+        <div className="relative w-full">
+          <label
+            htmlFor="email"
+            className="block font-bold mb-2 focus:outline-none"
+          >
+            Email*:
+          </label>
+          <input
+            name="email"
+            type="email"
+            id="email"
+            required
+            placeholder="Your email"
+            className={`block w-full rounded-md border-2 ${
+              !email ? 'border-blue-900' : ''
+            } placeholder:text-blue-500 placeholder:font-bold py-1 px-3 focus:outline-none ${
+              email && emailValid ? 'border-green-500' : ''
+            } ${email && !emailValid ? 'border-red-500' : ''}`}
+            onChange={(e) => {
+              setemail(e.target.value);
+            }}
+            onFocus={() => {
+              setemailFocus(true);
+            }}
+            onBlur={() => {
+              setemailFocus(false);
+            }}
+          />
+          {emailFocus ? <p>A valid and registered email is required</p> : ''}
+        </div>
+        <div className="flex flex-row justify-between">
+          <Button name={'Submit'} type={'submit'} />
+          <Link to="/">
+            <Button name={'Cancel'} type={'button'} icon={'cancel'} />
+          </Link>
+        </div>
+        <small className="bg-yellow-300 p-2 mt-5 block font-bold rounded-md">
+          To test navigation: Use "fab@me.com" as email
+        </small>
+      </form>
+    </>
+  );
+}
+
+export default ResetFormPage;
```

Comment:
try to increase button and text spacing since you mentioned that you will have to remove the yellow instruction section therefore with current spacing the card will become too short